_apps/validate-commit-msg-bot.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: validate-commit-msg-bot
+title: validate-commit-msg
 description: Runs validate-commit-msg over all commits of new or edited pull requests
   and sets an appropriate a status check.
-slug: validate-commit-msg-bot
+slug: validate-commit-msg
```

Comment:
Make sure you change this on GitHub, too — https://github.com/apps/validate-commit-msg currently 404s.

Reply:
@tlvince I was curious if you would be open to renaming your app to `validate-commit-msg`. There are 2 reasons for this request:

1. GitHub already shows a `[bot]` label next to any app in many places in the UI, so anything coming from this app will show up as `validate-commit-msg-bot[bot]`.
2. A "bot" is typically a personification of an app. Some apps present themselves as stand-ins for humans and should be called a bot, but it appears that `validate-commit-msg` falls more in the "algorithm" or "app" camp.

Reply:
Sure. Makes sense: https://github.com/tlvince/validate-commit-msg-bot/issues/6